packages/typespec-azure-portal-core/vitest.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    environment: "node",
+    isolate: false,
+    coverage: {
+      reporter: ["cobertura", "json", "text"],
+    },
+    outputFile: {
+      junit: "./test-results.xml",
+    },
+  },
+});
```

Comment:
```suggestion
import { defineConfig, mergeConfig } from "vitest/config";
import { defaultTypeSpecVitestConfig } from "../../core/vitest.workspace";

export default mergeConfig(defaultTypeSpecVitestConfig, defineConfig({}));
```

moved to use a shared config since then